### ISSUE TITLE
Add Pull Request Template and CHANELOG.md for updates in Github Workflow and Github App

### ISSUE THREAD

**Issue by BigSamu** [BigSamu/OpenSearch-bot#17]
OpenSearchp-bot is expected to grow. It would be nice to have a formal Pull Request template plus a CHANGELOG.md for having a formal procedure so people from the OpenSearch community can collaborate on this tool.

**Comment by BigSamu** [comment]
@JohnathonBowers,

This is ready? 

@ashwin-pc,

Should we create this for the beta release? What do you think?

**Comment by ashwin-pc** [comment]
Lets focus our efforts on release the app for now. Once we have it integrated with OSD, we can come back and tackle these issues :)

**Comment by BigSamu** [comment]
Agree. Let's keep this issue open and update the title of this issue.

The Pull Request Template and CHANGELOG.md should be added both for the [Changelog Workflow](https://github.com/BigSamu/OpenSearch_Parse_Changelog_Action) and the [Changelog PR Bridge](https://github.com/BigSamu/OpenSearch_Changeset_Bot) Services
